demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -683,14 +679,14 @@ class ContentList extends React.Component {
             }
           </div>
           <div className="choice-input-button-wrapper">
-            <div className="auto-play">
-              AutoPlay
+          <div className="auto-play">
+              Options
```

Comment:
Wouldn't `Show options` be better?

Because if I understand correctly, this toggle button is only displaying/hiding not enabling/disabling, right?

Reply:
Yes of course, forgot to edit it, should be done in the next commit.